nekoyume/Assets/_Scripts/UI/Mail.cs

```diff
@@ -178,22 +178,23 @@ private void UpdateMailList(long blockIndex)
             emptyImage.SetActive(!list.Any());
             UpdateTabs(blockIndex);
 
-            var tutorialController = Game.Game.instance.Stage.TutorialController;
-            var tutorialProgress = tutorialController.GetTutorialProgress();
-            if (tutorialProgress == 1)
-            {
-                var tutorialEquipment = States.Instance.CurrentAvatarState.mailBox
-                .FirstOrDefault(x =>
-                x is AttachmentMail mail &&
-                mail.requiredBlockIndex <= blockIndex &&
-                !(mail.attachment.itemUsable is null) &&
-                mail.attachment.itemUsable.ItemSubType == Nekoyume.Model.Item.ItemSubType.Weapon &&
-                mail.attachment.itemUsable.Id == TutorialEquipmentId);
-                if (!(tutorialEquipment is null))
-                {
-                    OnReceivedTutorialEquipment();
-                }
-            }
+            // Temporarily disable mail tutorial.
```

Comment:
Why did you comment out the code without deleting it?

Reply:
This code might be rollbacked later, but I think it would rather deleting it.